CONTRIBUTING.md

```diff
@@ -80,62 +114,57 @@ By viewing the "Files changed", you can leave a comment on any line of the diff.
 
-#### Every commit should work
+### Why do I have to sign a Contributor's Agreement?
 
-No individual commit should put the software in a broken state.
-It's fine for code to go unused (because the feature it's been introduced for isn't there yet), but no new functional issues should be introduced by a given commit.
-This ensures that reverting any individual commit from the branch is safe, and that git-bisect stays reliable for tracking down bugs.
+One of the key challenges for open source software is to support a collaborative development environment while protecting the rights of contributors and users over the long-term.
+Unifying Archivematica copyrights through contributor agreements is the best way to protect the availability and sustainability of Archivematica over the long-term as free and open-source software.
+In all cases, contributors who sign the Contributor's Agreement retain full rights to use their original contributions for any other purpose outside of Archivematica, while enabling Artefactual Systems, any successor Foundation which may eventually take over responsibility for Archivematica, and the wider Archivematica community to benefit from their collaboration and contributions in this open source project.
 
-#### Commit summaries should be concise
+[Artefactual Systems](http://artefactual.com) has made the decision and has a proven track record of making our intellectual property available to the community at large.
+By standardizing contributions on these agreements the Archivematica intellectual property position does not become too complicated.
+This ensures our resources are devoted to making our project the best they can be, rather than fighting legal battles over contributions.
 
-The commit summary (the first line of a commit message) should be short and clear.
-Usually, the first line of the commit message should be 50 - 80 characters.
-Since the summary is just a *summary* of the change, it should be readable at a glance when looking through git history.
+### How do I send in an agreement?
 
-A good commit summary makes it clear a) what *part* of the code is changing, and b) what the change is.
-For example:
+Please print out, read, sign, and scan the [contributor agreement](https://wiki.archivematica.org/images/2/25/Contributor_agreement.txt) and email it to agreement@artefactual.com
+Alternatively, you may send an original signed agreement to:
 
-Clear commit summary:
-> normalize: fall back to default rule for unidentified files
+    Artefactual Systems Inc.
+    201 - 301 Sixth Street
+    New Westminster BC  V3L 3A7
+    Canada
 
-Unclear commit summaries:
-> Fixed some normalization bugs
+### Small changes
 
-> Bugfixes
+Small contributions such as fixing spelling errors, where the content is small enough to not be considered intellectual property, do not require signing a CLA.
```

Comment:
I'm going to remove this suggested change, since the rest of this PR is ready to go. We can always decide later that we want the small-contributions exception to the contributor's agreement.